### ISSUE TITLE
High DPI support on Mac OS X

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet#476]
Someone with either of these OSes should look into fixing it.

**Comment by def-** [comment]
I've been trying to fix this and failed spectacularly. Apparently I can't guess correctly what Windows will do. Someone with a Windows computer should look into fixing this please.

**Comment by Jupeyy** [comment]
What is the behaviour under MacOS?
Does it still zoom? I researched for dpi awareness, and saw that SDL might provide a solution by setting
`SDL_SetHint(SDL_HINT_VIDEO_HIGHDPI_DISABLED, "1")` before SDL is initialized. That would ignore the DPI scale, since the client does it anyway by the projection matrix.

If that doesn't work, i can still disable the "zoom", but it will look like the resolution was lowered, since SDL can't create a window(the framebuffer of it) as full resolution. I tested it locally when i fixed it for windows(so as a workaround it might help).
